src/ScrumBoardItBundle/Services/AbstractApi.php

```diff
@@ -64,6 +68,44 @@ protected function call($url)
         return json_decode($content);
     }
 
+    protected function send($url, $content, $nbArguments)
+    {   
+        $ch = curl_init();
+        curl_setopt($ch, CURLOPT_URL, $url);
+        curl_setopt($ch, CURLOPT_HTTPHEADER, [
+            'Authorization: Basic ' . $this->getUser()->getHash() . "\naccept: application/json
+            \naccept-language: en-US,en;q=0.8
+            \nuser-agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/49.0.2623.108 Chrome/49.0.2623.108 Safari/537.36"
+        ]);
+        curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
+        curl_setopt($ch, CURLOPT_TIMEOUT, 5);
+        curl_setopt($ch, CURLOPT_POST, $nbArguments);
+        curl_setopt($ch, CURLOPT_POSTFIELDS, $content);
+        curl_exec($ch);
+        
+        curl_close($ch);
+    }
+
+    /**
+     * Initialize filters in session variable
+     *
+     * @param Session $session            
+     */
+    protected function initFilters(Session $session)
+    {
+        $session->set('filters', array(
+            'project' => null,
+            'sprint' => null
+        ));
```

Comment:
Are you sure, it's relevant to use the session var ?
If it's used to reload the page after printing the tickets, you should maybe just simply reload the page with GET parameters present.